pkg/printers/internalversion/printers.go

```diff
@@ -41,7 +41,6 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
```

Comment:
Let's not do that - that is how the regression happened in the first place!  Let's get this all fixed up first...